### DISCUSSION TITLE
Simulator not working on Mac M1

### DISCUSSION THREAD

**Discussion by emircanerkul** [zepp-health/zeppos-forum#139] [Developer Feedback]
I followed instruction but Still simulator gives error.

`[firmware_simulator]: chmod: Unable to change file mode on qemu_mac/qemu-system-arm: Operation not permitted`

---
After the installation is complete, execute the following command in the terminal to add executable permissions to the simulator startup script.

`cd /Applications/simulator.app/Contents/Resources/firmware && chmod +x ./start_qemu.sh`

Re-open a terminal and run the following command to start the simulator.

`cd /Applications/simulator.app/Contents/MacOS && sudo -s ./simulator`

---

<img width="1038" alt="image" src="https://user-images.githubusercontent.com/11430795/231693981-5fb7efe9-2a85-4aae-90f4-0404bbd8001f.png">

**Comment by MyoungXUE** [comment]
Thank you for your feedback, I will feedback this issue to the relevant colleagues.

**Comment by MyoungXUE** [comment]
Hi, please refer to this document for operation and retry afterwards.

https://docs.zepp.com/docs/guides/faq/apple-silicon/

**Comment by emircanerkul** [comment]
@MyoungXUE Hi, I already did these steps and enabled tunnelblick extension and I able to see tap0 - tap15

-- nevermind, I just reverted my system back it's a quite big security problem, and too much trust in non-open source software.

**Comment by emircanerkul** [comment]
I used https://codeberg.org/Freeyourgadget/Gadgetbridge
